test/OpenTelemetry.Instrumentation.Http.Tests/HttpTestData.cs

```diff
@@ -49,11 +49,7 @@ public static string NormalizeValues(string value, string host, int port)
         return value
             .Replace("{host}", host)
             .Replace("{port}", port.ToString())
-#if NETFRAMEWORK
             .Replace("{flavor}", "1.1");
-#else
-            .Replace("{flavor}", "2.0");
```

Comment:
This is kind of interesting. How is it we don't report 2.0 now?

Reply:
@CodeBlanch See https://github.com/open-telemetry/opentelemetry-dotnet/issues/4928